app/products/calls/connection/websocket_client.ts

```diff
@@ -99,6 +101,7 @@ export class WebSocketClient extends EventEmitter {
                 }
                 return;
             } else if (!this.connID) {
+                logWarning('calls: ws message received while waiting for hello');
```

Comment:
do we really want to log a warning here or is this more of a debug information?

Reply:
yep, good point, changed to debug, thanks!